mm2src/coins/Cargo.toml

```diff
@@ -173,5 +172,5 @@ mm2_test_helpers = { path = "../mm2_test_helpers" }
 wagyu-zcash-parameters = { version = "0.2" }
 
 [build-dependencies]
-prost-build = { version = "0.10.4", default-features = false }
-tonic-build = { version = "0.7", default-features = false, features = ["prost", "compression"] }
+prost-build = { version = "0.11", default-features = false }
```

Comment:
Can you bump this one to the same version too? https://github.com/KomodoPlatform/komodo-defi-framework/blob/00844ca6f79a24fe5271f8ffb4f00c0f63611801/mm2src/mm2_main/Cargo.toml#L132

Reply:
oh thanks, that would hopefully help reduce duplications.